cocos/platform/ohos/CCGLViewImpl-ohos.cpp

```diff
@@ -0,0 +1,188 @@
+#include "platform/CCPlatformConfig.h"
+#if CC_TARGET_PLATFORM == CC_PLATFORM_OHOS
+#include <stdlib.h>
+#include "CCGLViewImpl-ohos.h"
+#include "base/CCDirector.h"
+#include "base/ccMacros.h"
+#include "base/CCIMEDispatcher.h"
+#include "napi/helper/Js_Cocos2dxHelper.h"
+#include "CCGL-ohos.h"
+#include "CCLogOhos.h"
+#include "napi/helper/NapiHelper.h"
+
+
+
+//#if CC_TEXTURE_ATLAS_USE_VAO
+#include <EGL/egl.h>
+PFNGLGENVERTEXARRAYSOESPROC glGenVertexArraysOESEXT = 0;
+PFNGLBINDVERTEXARRAYOESPROC glBindVertexArrayOESEXT = 0;
+PFNGLDELETEVERTEXARRAYSOESPROC glDeleteVertexArraysOESEXT = 0;
```

Comment:
It's better to use `nullptr` to initialize a pointer variable.

Reply:
same with android